main.cpp

```diff
@@ -180,7 +172,7 @@ void wait (SDL_Surface *screen, SDL_Rect *blockPosition, int *score,bool *done,b
             testDirection(screen,blockPosition,&Right,&Left,&Up,&Down);
             switch(randomMove){
 
-                case 1 : //move to the Right then invert
+                case 1 : //move to the Right
                 while (Right==true && *blockCatch == false && *done == false ){
```

Comment:
`Right` should be `right`! 
Also could ` while (Right==true && *blockCatch == false && *done == false )` be simplified to `  while (Right && *blockCatch == false && *done == false )` ?